src/main/java/org/spongepowered/common/mixin/core/server/MixinMinecraftServer.java

```diff
@@ -791,4 +795,10 @@ public Object onCall(Callable<?> callable) throws Exception {
     private Object onRun(FutureTask<?> task, Logger logger) {
         return SpongeImplHooks.onUtilRunTask(task, logger);
     }
+
+    @Inject(method = "addServerInfoToCrashReport", at = @At("RETURN"), cancellable = true)
+    public void onCrashReport(CrashReport report, CallbackInfoReturnable<CrashReport> cir) {
```

Comment:
You can make this injection method private, there is no need to override it in subclasses.